sklearn/preprocessing/data.py

```diff
@@ -430,6 +440,126 @@ def inverse_transform(self, X, copy=None):
         return X
 
 
+class MaxAbsScaler(BaseEstimator, TransformerMixin):
+    """Scale each feature to the [-1, 1] range without breaking the sparsity.
```

Comment:
I would maybe reformulate that. The min and max will not be -1 and 1. Scale features to lie within [-1, 1] range without shift?